consent/types_test.go

```diff
@@ -0,0 +1,79 @@
+package consent
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/ory/fosite"
+	"github.com/stretchr/testify/require"
+)
+
+func TestToRFCError(t *testing.T) {
+	for k, tc := range []struct {
+		input  *RequestDeniedError
+		expect *fosite.RFC6749Error
+	}{
+		{
+			input: &RequestDeniedError{
+				Description: "not empty",
+			},
+			expect: &fosite.RFC6749Error{
+				Name:        "",
```

Comment:
Wouldn't it make sense for the name and description to always be set to the default values? Then you can override e.g. just the description or only the name without having to set both?